proposals/lip-0048.md

```diff
@@ -25,15 +25,34 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
 
 ## Rationale
 
+The fee structure presented here serves to allow defining sufficient cost to manage state growth and prevent transaction pool spamming. In principle, any transaction resulting in a state change should incur a fee. This reasoning is implemented in the `payFee` method, which can be invoked by other modules.
```

Comment:
I don't understand how the fee prevents transaction pool spamming, can you elaborate a bit?

Reply:
I'm not sure either, I believe I picked this up in some discussions related to fees. Please suggest how to rephrase it.

Reply:
I think I understand now what your question implied, will try to correct it.